clang-tools-extra/docs/clang-tidy/checks/readability/redundant-inline-specifier.rst

```diff
@@ -0,0 +1,25 @@
+.. title:: clang-tidy - readability-redundant-inline-specifier
+
+readability-redundant-inline-specifier
+======================================
+
+Checks for instances of the ``inline`` keyword in code where it is redundant
```

Comment:
Please synchronize with statement in Release Notes.

Reply:
I apologies, but I'm not quite sure to understand what you're asking me to do here? Would you be able to give me more info?

Reply:
Just make first statement in documentation and statement in Release Notes same.

Reply:
Right, I was looking at the wrong file hence why I was confused.